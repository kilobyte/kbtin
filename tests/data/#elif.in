#ifali foo {#show wrong} #elif 1 {#show #ifaliasexists ✓}
#ifali foo {#show wrong} #elif 0 {#show #ifaliasexists ✓} #else {#show #ifaliasexists ✗}
#ife foo {#show wrong} #elif 1 {#show #ifexists ✓}
#ife foo {#show wrong} #elif 0 {#show #ifexists ✓} #else {#show #ifexists ✗}
#grep foo bar {#show wrong} #elif 1 {#show #grep ✓}
#grep foo bar {#show wrong} #elif 0 {#show #grep ✓} #else {#show #grep ✗}
#strcmp foo bar {#show wrong} #elif 1 {#show #strcmp ✓}
#strcmp foo bar {#show wrong} #elif 0 {#show #strcmp ✓} #else {#show #strcmp ✗}
