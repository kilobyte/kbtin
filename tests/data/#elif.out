#ifaliasexists ✓
#ifaliasexists ✗
#ifexists ✓
#ifexists ✗
#grep ✓
#grep ✗
#strcmp ✓
#strcmp ✗
